### PR TITLE
News on UFR RDM policy update

### PR DESCRIPTION
**Pull request by sebastian-schaaf**
University of Freiburg with a new policy on Research Data Management

### CODE REVIEW

**Review by JanLeendertse — COMMENTED** [review]
Text passt für mich. Danke!

File: content/news/2023-11-16-uni-freiburg-new-rdm-policy/index.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+title: 'RDM policy at University of Freiburg'
+date: "2023-11-16"
+tease: "New Research Data Management policy replaces previous version"
+hide_tease: false
+subsites: [all-eu]
+main_subsite: eu
```
Comment (bgruening) [inline content/news/2023-11-16-uni-freiburg-new-rdm-policy/index.md:7]:
```suggestion
main_subsite: freiburg
```

File: content/news/2023-11-16-uni-freiburg-new-rdm-policy/index.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+---
+title: 'RDM policy at University of Freiburg'
+date: "2023-11-16"
+tease: "New Research Data Management policy replaces previous version"
+hide_tease: false
+subsites: [all-eu]
```
Comment (bgruening) [inline content/news/2023-11-16-uni-freiburg-new-rdm-policy/index.md:6]:
```suggestion
subsites: [freiburg]
```

File: content/news/2023-11-16-uni-freiburg-new-rdm-policy/index.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+---
+title: 'RDM policy at University of Freiburg'
+date: "2023-11-16"
+tease: "New Research Data Management policy replaces previous version"
```
Comment (bgruening) [inline content/news/2023-11-16-uni-freiburg-new-rdm-policy/index.md:4]:
```suggestion
tease: "New Research Data Management policy for the University of Freiburg"
```

**Comment by sebastian-schaaf** [comment]
Suggestions accepted - could be published I guess

**Comment by bgruening** [comment]
Please take it next time out of Draft mode.

**Comment by bgruening** [comment]
Thanks!

**Pull request merged**